packages/fluentui/react-builder/src/components/ReaderNarration.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Alert, Ref, Dropdown, DropdownProps } from '@fluentui/react-northstar';
+import { IAriaElement } from './../narration/NarrationComputer';
+import { DescendantsNarrationsComputer } from './../narration/DescendantsNarrationsComputer';
+
+const computer: DescendantsNarrationsComputer = new DescendantsNarrationsComputer();
+let narrationTexts: Record<string, string> = {};
+
+export type ReaderNarrationProps = {
+  selector: string;
+};
+
+export const ReaderNarration: React.FunctionComponent<ReaderNarrationProps> = ({ selector }) => {
+  const ref = React.useRef<HTMLElement>();
+  const [narrationText, setNarrationText] = React.useState('');
+  const [narrationPath, setNarrationPath] = React.useState('');
+  const [narrationPaths, setNarrationPaths] = React.useState([]);
+
+  React.useEffect(() => {
+    if (ref.current) {
+      // Begin if 1
+      const element = ref.current.ownerDocument.querySelector(selector) as IAriaElement;
+
+      // Compute and store the narrations for the element and its focusable descendants
+      computer.compute(element, 'Win/JAWS').then(narrations => {
+        const paths: string[] = [];
+        narrationTexts = {};
+        narrations.forEach(narration => {
+          // Begin forEach 1
+          const path = narration.path.join(' > ');
+          paths.push(path);
+          narrationTexts[path] = narration.text;
+        }); // End forEach 1
+
+        // Update the narration paths dropdown values
+        setNarrationPaths(paths);
+
+        // If some narration has been retrieved, choose the first one as the narration to be displayed
+        const text = narrations[0]?.text || null;
+        setCompleteText(text);
+      }); // End compute
+    } // End if 1
+  }, [setNarrationText, setNarrationPath, setNarrationPaths, ref, selector]);
+
+  // Composes and sets the complete screen reader narration text to be displayed
+  const setCompleteText = text => {
+    setNarrationText(`Narration: ${text}`);
+  }; // End setCompleteText
+
+  const handleNarrationPathChange = (event: any, props: DropdownProps) => {
```

Comment:
You don't seem to use the event, will setting it to an anonymous variable work ? 

```suggestion
  const handleNarrationPathChange = (_: any, props: DropdownProps) => {
```

Reply:
What are the advantages of using `_` instead? If it is just a matter of code style, I would prefer a variable with a meaningful name.

Reply:
That's fair, it's a personal preference for me to indicate that this variable is not being used. Feel free to ignore or ask team members who have been here longer